src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Info/DiaryInfoViewModel.swift

```diff
@@ -25,7 +25,6 @@ struct DiaryInfoViewModel {
 						UIImage(
 							imageLiteralResourceName: "Illu_ContactDiary-Information"
 						),
-						accessibilityLabel: AppStrings.ContactDiary.Information.imageDescription,
```

Comment:
Is the string used somewhere else or could it be removed from the `AppStrings`?

Reply:
AccessibilityIdentifiers.ContactDiaryInformation.imageDescription points to it
But that one is only used here, shall we remove it too?

Reply:
But accessibilityIdentifier is required for .image cells

Reply:
I think the AccessibilityIdentifiers don't actually use the AppStrings. They just use "AppStrings." as prefix which is a bit confusing. I think we could change that to make this more clear and e.g. use "AccessibilityIdentifier." as a prefix. But for this PR I think it's save to just delete the string that is not used anymore from the Localizable.strings file.

Reply:
@nickguendling Good point, I#ll remove them

Reply:
I just took another look, I think the JIRA ticket is probably referring to the book illustration on the Home Screen, not the illustration on top of the information view controller. Because I think the hand holding a mobile, that is also mentioned, is referring to the test result/submission cell that also does not have an accessibility label 🤔